apps/indexer/lib/indexer/transform/token_transfers.ex

```diff
@@ -82,7 +82,7 @@ defmodule Indexer.Transform.TokenTransfers do
 
   @doc "Discerns CELO token transfers from internal transactions"
   def parse_itx(txs, gold_token) do
-    initial_acc = %{token_transfers: [], gold_token: gold_token, transfer_count: 0}
+    initial_acc = %{token_transfers: [], gold_token: gold_token, transfer_count: 1}
```

Comment:
It's not super straightforward why we initialize `transfer_count` with `1` here. In order to understand the intent, you need to check a different function. Maybe it makes sense to leave `0` here but change the line where we assign `log_index` below? Like `log_index: -1 * (transfer_count + 1)` and add a comment that clarifies the intent?